packages/devextreme/js/__internal/core/license/license_validation.ts

```diff
@@ -1,3 +1,6 @@
+import errors from '@js/core/errors';
+import { version } from '@js/core/version';
```

Comment:
Minor comment.
(You can apply it or not by your opinion)

Let's change naming here
```suggestion
import { version as packageVersion } from '@js/core/version';
```

To fix rename ```ver``` -> ```version``` in the ```verifyLicense ``` function.